cloud_service/upgrading-cloud/upgrade-cloudsvc-roxctl.adoc

```diff
@@ -0,0 +1,68 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="upgrade-cloudsvc-roxctl"]
+= Manually upgrading secured clusters in RHACS Cloud Service by using the `roxctl` CLI
+include::modules/common-attributes.adoc[]
+:context: upgrade-cloudsvc-roxctl
+
+toc::[]
+
+[role="_abstract"]
+You can upgrade your secured clusters in {product-title-managed-short} by using the `roxctl` CLI.
+
+[IMPORTANT]
+====
+You need to perform the manual upgrade procedure only if you used the `roxctl` CLI to install the secured clusters.
+====
+
+[id="upgrade-roxctl-cli"]
+== Upgrading the `roxctl` CLI
+
+To upgrade the `roxctl` CLI to the latest version you must uninstall the existing version of `roxctl` CLI and then install the latest version of the `roxctl` CLI.
+
+include::modules/uninstall-roxctl-cli.adoc[leveloffset=+2]
+include::modules/install-roxctl-cli-linux.adoc[leveloffset=+2]
+include::modules/install-roxctl-cli-macos.adoc[leveloffset=+2]
+include::modules/install-roxctl-cli-windows.adoc[leveloffset=+2]
+
+[id="upgrade-secured-clusters-cloud"]
+== Upgrading all secured clusters manually
+
+[IMPORTANT]
```

Comment:
I'm not sure why this admonition is separated into nested lists. In the preview, it makes the red box take up a lot of the page. Consider revising the wording to something like this to make the page easier to scan:
`* If you are using automatic upgrades, update all your secured clusters by using automatic upgrades.`
`* If you are not using automatic upgrades, complete the instructions in this section on all secured clusters. To ensure optimal functionality, use the same {product-title-short} version for your secured clusters that {product-title-managed-short} is running.`